### DISCUSSION TITLE
Possible to sync only a subfolder?

### DISCUSSION THREAD

**Discussion by gerroon** [vrtmrz/obsidian-livesync#332] [Q&A] [answered]
Hi

Is there a way to sync a subfolder of the Obsidian notebook say "mynotes/mobile" ? That way I do not need to sync the whole notebook folder with my mobile devices and yet have my notes from mobile avail?

thanks

**Comment by vrtmrz** [comment, marked as answer]
Hello! Thank you for asking me!
We can restrict the synchronising files by `Regular expressions for restricting synchronisation target` in the `Sync setting` pane.
In this case, we can set `^mynotes\/mobile` for it.
Would you please try this once?

**Reply by denny0810** [reply]
Hello!
I faced the same problem that just want to sychronize "!Notebook/dailyNotes" folder with my mobile. I tried to set `^!Notebook\/dailyNotes` as suggested and then I fetched from my server. Unfortunately, it seems to fetch all the contents instead of what I expected. 
I'm not familiar with the Reg expression.  How should I correct? Is it possible to design an option to select the folders/files that want?
